docs/user_docs/concepts/intent

```diff
@@ -0,0 +1,90 @@
+# Intent
+
+Intent represents the operational intentions you want to deliver with Kusion. The intentions here is expected to contain all components during the SDLC, such as resources, dependencies and policies. All AppConfigurations written by application developers will also be converted into the Intent by Kusion module generators before Kusion acutally perform operations.
+
+## Purpose
+
+### Single Source of Truth
+
+In Kusion's workflow, platform engineer build Kusion modules and provide environment configurations, application developers choose Kusion modules they need and deploy operational intentions to an environment with related environment configurations. They can also input dynamic paramters like the container image when execute the `kusion build` command. So the final operational intentions includes configuratins written by application developers, environment configurations and dynamic inputs. Due to this reason, we introduce **Intent** to represent the SSoT(Single Source of Truth) of Kusion. It is the result of `kusion build` which contains all operational intentions from different sources.
+
+### Consistency
```

Comment:
> This explicit intent also allows us to introspect the configuration, which makes troubleshooting complex setups easier. 
imho Intent is an unified model, not only consistency we also need to think about the benefits of unified model.